packages/nx/src/plugins/target-defaults/target-defaults-plugin.ts

```diff
@@ -16,13 +16,17 @@ import {
 import { getGlobPatternsFromPackageManagerWorkspaces } from '../package-json-workspaces';
 
 /**
- * This symbol marks that a target provides information which should modify a target already registered
+ * This marks that a target provides information which should modify a target already registered
  * on the project via other plugins. If the target has not already been registered, and this symbol is true,
  * the information provided by it will be discarded.
  */
-export const ONLY_MODIFIES_EXISTING_TARGET = Symbol(
-  'ONLY_MODIFIES_EXISTING_TARGET'
-);
+export const ONLY_MODIFIES_EXISTING_TARGET = 'NX_ONLY_MODIFIES_EXISTING_TARGET';
+
+/**
+ * This symbol is used to override the source file for the target defaults plugin.
```

Comment:
Nit: missed reference to symbol.

Nit 2: might be nice to add a comment quickly referencing why we don’t use an actual symbol so that nobody tries to be “helpful” in future be make it an actual one :)